pkg/cli/add_vcluster_helm.go

```diff
@@ -44,6 +48,46 @@ func AddVClusterHelm(
 		return err
 	}
 
+	// If the vCluster was paused with the helm driver, adding it to the platform will only create the secret for registration
+	// which leads to confusing behavior for the user since they won't see the cluster in the platform UI until it is resumed.
+	if vCluster.Annotations != nil && vCluster.Annotations[constants.PausedAnnotation] == "true" {
+		answer, err := log.Question(&survey.QuestionOptions{
+			Question:     fmt.Sprintf("vCluster %s is asleep and needs to be reawakened before it can be added.  Would you like to wake it up and add it now?", vClusterName),
+			DefaultValue: "no",
+			Options: []string{
+				"no",
+				"yes",
+			},
+		})
+
+		if err != nil {
+			return fmt.Errorf("failed to capture your repsponse %w", err)
+		}
+
+		// Bail if they don't want to wake the vCluster yet.
+		if answer == "no" {
+			return fmt.Errorf("please wakeup vCluster %s before adding it to the platform", vClusterName)
```

Comment:
This seems a little harsh... like they chose the wrong answer. Should we log a message like "The vCluster will be added the next time it is active."?

Reply:
I said please... :)